website_sale_require_legal/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Require accepting legal terms to checkout",
+    "summary": "Force the user to accept legal tems to buy in the web shop",
+    "version": "8.0.1.0.0",
+    "category": "Website",
+    "website": "https://odoo-community.org/",
+    "author": "Antiun Ingeniería, S.L., Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
+        "python": [],
+        "bin": [],
+    },
+    "depends": [
+        "website_legal_page",
```

Comment:
This module depends on website_legal_page, no incompatibility possible between them. Runbot works for me, please elaborate steps to get to error.